feat: renamed interfaces

polomani opened the pull request:
BREAKING CHANGE:

Interfaces renamed:
- AzureADConfigurationDTO -> AzureADConfiguration
- DatapointsGetAggregateDatapoint -> DatapointsAggregates
- DatapointsGetDatapoint -> Datapoints
- DatapointsGetDoubleDatapoint -> DoubleDatapoints
- DatapointsGetStringDatapoint -> StringDatapoints
- DatapointsInsertProperties -> ExternalDatapoints
- DatapointsPostDatapoint -> ExternalDatapointsQuery
- ExternalFilesMetadata -> ExternalFileInfo
- FilesMetadata -> FileInfo
- GetAggregateDatapoint -> DatapointAggregate
- GetDatapointMetadata -> DatapointMetadata
- GetDoubleDatapoint -> DoubleDatapoint
- GetStringDatapoint -> StringDatapoint
- GetTimeSeriesMetadataDTO -> TimeSeries
- OAuth2ConfigurationDTO -> OAuth2Configuration
- PostDatapoint -> ExternalDatapoint
- TimeSeriesSearchDTO -> TimeSeriesSearchFilter
- TimeseriesFilter -> TimeseriesFilterQuery (structure changed)
- TimeseriesFilterProps -> TimeseriesFilter
- UploadFileMetadataResponse -> FileUploadResponse

Removed interfaces:
- Filter (use TimeseriesFilter)
- Search (use TimeseriesSearch)

Changed:
- `timeseries.list()` signature is now consistent with other resource types

haved reviewed (approved):
LGTM!
Including timeseries list changes

polomani committed:
feat: interface changes

BREAKING CHANGE:

Interfaces renamed:
- AzureADConfigurationDTO -> AzureADConfiguration
- DatapointsGetAggregateDatapoint -> DatapointsAggregates
- DatapointsGetDatapoint -> Datapoints
- DatapointsGetDoubleDatapoint -> DoubleDatapoints
- DatapointsGetStringDatapoint -> StringDatapoints
- DatapointsInsertProperties -> ExternalDatapoints
- DatapointsPostDatapoint -> ExternalDatapointsQuery
- ExternalFilesMetadata -> ExternalFileInfo
- FilesMetadata -> FileInfo
- GetAggregateDatapoint -> DatapointAggregate
- GetDatapointMetadata -> DatapointMetadata
- GetDoubleDatapoint -> DoubleDatapoint
- GetStringDatapoint -> StringDatapoint
- GetTimeSeriesMetadataDTO -> TimeSeries
- OAuth2ConfigurationDTO -> OAuth2Configuration
- PostDatapoint -> ExternalDatapoint
- TimeSeriesSearchDTO -> TimeSeriesSearchFilter
- TimeseriesFilter -> TimeseriesFilterQuery (structure changed)
- TimeseriesFilterProps -> TimeseriesFilter
- UploadFileMetadataResponse -> FileUploadResponse

Removed interfaces:
- Filter (use TimeseriesFilter)
- Search (use TimeseriesSearch)

Changed:
- `timeseries.list()` signature is now consistent with other resource types

polomani committed:
feat: interfaces changed

TimeSeries -> Timeseries
TimeSeriesImpl -> TimeseriesImpl
ExternalTimeSeries -> ExternalTimeseries
TimeSeriesSearchFilter -> TimeseriesSearchFilter
TimeSeriesSearch -> TimeseriesSearch

polomani committed:
feat: renamed interfaces

DatapointMetadata -> DatapointInfo
